src/reducers/registry.js

```diff
@@ -0,0 +1,28 @@
+// @flow
+import { REGISTRY } from "../constants/actionTypes";
+import { type RegistryActions } from "../actions/registry";
+
+export type State = {
+  +error: string,
+  +loading: boolean,
+  +result: string
+};
+const initialState: State = { error: "", result: "", loading: false };
+
+const registryReducer = (
+  state: State = initialState,
+  action: RegistryActions
+): State => {
+  switch (action.type) {
+    case REGISTRY.REQUESTED:
+      return { error: "", result: "", loading: true };
```

Comment:
add '...state' here. Just to keep code consistent.
Take it in the habit, to avoid potential bugs in the future.